x-pack/plugins/alerting_builtins/server/alert_types/index_threshold/lib/time_series_query.ts

```diff
@@ -94,6 +94,7 @@ export async function timeSeriesQuery(
     dateAgg: {
       date_range: {
         field: timeField,
+        format: 'strict_date_time',
```

Comment:
This was added to indicate the format of the dates that we pass in as the date range values.  Without it, an error will occur if the dates in the index are **not** in this format, eg `epoch_millis`.